Formula/amazon-efs-utils.rb

```diff
@@ -216,6 +216,11 @@ def caveats
         sudo rm /Library/Filesystems/efs.fs/Contents/Resources/mount_efs
 
       To enable watchdog for TLS mounts:
+        * Prerequisite for mac2.metal *
+          Mac2 has different installation path /opt/homebrew comparing with Mac1, run following commands to update the paths in watchdog.plist to use this
+          VERSION=<e.g. 1.34.5>
+          sed s@/usr/local/@/opt/homebrew/@g /opt/homebrew/Cellar/amazon-efs-utils/${VERSION}/libexec/amazon-efs-mount-watchdog.plist
```

Comment:
Suggestion: remove environment variable in favor of ruby `version` variable.

```suggestion
          sed s@/usr/local/@/opt/homebrew/@g /opt/homebrew/Cellar/amazon-efs-utils/#{version}/libexec/amazon-efs-mount-watchdog.plist
```

With this change, Homebrew will output the caveats like so:
```
 To enable watchdog for TLS mounts:
   * Prerequisite for mac2.metel - run following commands to edit paths in plist
     sed s@/usr/local/@/opt/homebrew/@g /opt/homebrew/Cellar/amazon-efs-utils/1.34.5/libexec/amazon-efs-mount-watchdog.plist

  sudo cp /usr/local/opt/amazon-efs-utils/libexec/amazon-efs-mount-watchdog.plist /Library/LaunchAgents
  sudo launchctl load /Library/LaunchAgents/amazon-efs-mount-watchdog.plist
```